app/shared-components/PremiumPromoModalContent/PremiumPromoModalContent.jsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * Premium Promo Modal Component
+ *
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2019 Ghostery, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
+
+/**
+ * A functional React component for a Premium Promo Modal that may be displayed in the Hub and/or Panel
+ * @return {JSX} JSX for rendering a Premium Promo Modal
+ * @memberof SharedComponents
+ */
+const PremiumPromoModalContent = (props) => {
+	const {
+		isPlus,
+		handleTryMidnightClick,
+		handleGetPlusClick,
+		handleKeepBasicClick,
+		handleGoAwayClick,
+		location,
+		XButton
+	} = props;
+
+	const contentClassNames = ClassNames(
+		'PremiumPromoModal__content',
+		'flex-container',
+		'flex-dir-column',
+		'align-middle',
+	);
```

Comment:
Also, you don't need to use the `classnames` library if none of your class assignments are dependent on logic.